scripts/ci

```diff
@@ -55,11 +60,21 @@ function build() {
 }
 
 function ci() {
-    build $1
+    build $1 $2
     command_tests
 }
 
-if [ "${CI_CHECK_FORMAT}" == "1" ] ; then
+build_type=$1
+full_build=$2
+if [ "" == "${full_build}" ] ; then
+    if [ "false" == "${TRAVIS_PULL_REQUEST}" ] || [ "" == "${TRAVIS_PULL_REQUEST}" ] ; then
+        full_build=0
+    else
+        full_build=1
+    fi
+fi
+
+if [ "1" == "${CI_CHECK_FORMAT}" ] ; then
```

Comment:
Likewise, let's switch the condition to `"${CI_CHECK_FORMAT}" == "1"`

Reply:
Changed the order